pageserver/src/tenant/ephemeral_file.rs

```diff
@@ -36,12 +36,15 @@ pub struct EphemeralFiles {
 }
 
```

Comment:
A comment to explain the new scheme would be nice. Something like "An ephemeral file is append-only. We keep the last page, which can still be modified, in `mutable_head`. The other pages, which can no longer be modified, are accessed through the page cache".